setup.cfg

```diff
@@ -33,7 +35,7 @@ python_requires = >=3.6
 install_requires =
     networkx~=2.2
     tex2pix~=0.3
-    particle~=0.14
+    particle~=0.16
```

Comment:
Why does particle need to be updated? These should be lower bounds on the `pylhe` side.

Reply:
I know you mentioned that you wanted to "ensure that PDG 2021", but we shouldn't need to update lower bounds on this, right? You haven't yanked older releases from PyPI as there isn't anything "wrong" with the older releases. I guess I'm not sure how to think about the versioning for `particle`. Should we think of the minor releases as being more like CalVer then SemVer?

Reply:
> Why does particle need to be updated? These should be lower bounds on the `pylhe` side.

Indeed you are largely shadowed from most developments in Particle since only dependent for functionality that is very stable. On the other hand Particle does evolve, with PDG updates, the odd fix, etc., and dependents should evolve accordingly IMO. For example at some point I also updated the dependency on Lark in Particle. But up to you. I will remove the change in the dependency since anyway PRs should be scoped and I'm doing 2 things here.

Reply:
Right, nothing wrong with older versions (modulo the odd buglet that is caught rarely but once in a while). No need to yank any release so far.

I do not use CalVer. I work more lile SemVer. So far there's never been a major 1.0 release because there are still some things I would like to freeze first in the API/package. As for minor version updates they mostly take place when moving to new data files (0.16 got out with the PDG 2021 data, for example); BTW, data files have a version too, just in case. Minor version changes also take place when fixes are needed to particles properties, as that kind of means a different behaviour since some methods get output changed (case when there was a bug in the parity of antibaryons). I hope this makes sense. (Probably more than you really wanted to ask :D.)

Reply:
This PR is hence ready.

Reply:
Sorry I missed these! But thank you for the info, Eduardo!